pkg/verifier/gcp/entry_point.go

```diff
@@ -6,14 +6,16 @@ import (
 	"math/rand"
 	"strconv"
 
+	"github.com/openshift/osd-network-verifier/pkg/data/cpu"
+	"github.com/openshift/osd-network-verifier/pkg/data/egress_lists"
+	"github.com/openshift/osd-network-verifier/pkg/helpers"
 	"github.com/openshift/osd-network-verifier/pkg/output"
-	"github.com/openshift/osd-network-verifier/pkg/probes/dummy"
+	"github.com/openshift/osd-network-verifier/pkg/probes/curl"
 	"github.com/openshift/osd-network-verifier/pkg/verifier"
 )
 
 const (
-	cloudImageIDDefault   = "rhel-9-v20240703"
-	DEFAULT_INSTANCE_TYPE = "e2-micro"
+	DEFAULT_TIMEOUT = 5
```

Comment:
```suggestion
	DEFAULT_TIMEOUT = 5 * time.Second
```
You'll also have to import `"time"` (if your IDE doesn't add the import for you automatically)